packages/loctool/package.json

```diff
@@ -91,10 +93,11 @@
         "xml-js": "^1.6.11"
     },
     "devDependencies": {
+        "@jest/types": "^29.6.3",
```

Comment:
I think you meant `@jest/globals` here, however to get those types automatically with no additional configuration it's easier to use `@types/jest`

Reply:
Fixed